docs/platform-services/automation-service/app-central/integrations/aws-iam.md

```diff
@@ -42,5 +52,9 @@ Using the integration with IAM, you can create and manage AWS users and groups,
 * October 1, 2019 - First upload
 * June 21, 2023 (v1.1) - Updated the integration with Environmental Variables
 * January 16, 2024 (v1.2)
-	+ Changed the field type of the Access key to password
-	+ Updated action: Get User (Table View issue fixed)
+    + Changed the field type of the Access key to password
+    + Updated action: Get User (Table View issue fixed)
+* January 24, 2024 (v1.3)
+    + Added New Action: Update Access Key
+* February 5, 2024 (v1.4)
+    + Added New Action: Get Access Key Last Used
```

Comment:
```suggestion
    + Added New Action: Get last used Access Key
```

Reply:
@JV0812, this should stay as-is since it is the name of an actual action in the integration.